socketcan_bridge/src/socketcan_to_topic.cpp

```diff
@@ -55,6 +55,7 @@ namespace socketcan_bridge
       can_topic_ = nh->advertise<can_msgs::Frame>("received_messages",
                                                   nh_param->param("received_messages_queue_size", 10));
       driver_ = driver;
+      timer = nh->createTimer(ros::Duration(0.1), std::bind(&SocketCANToTopic::timerCallback, this, std::placeholders::_1));
```

Comment:
This is a polling approach. We should at least be able to configure the duration via for example a ROS parameter. Also there should be an option to disable this behaviour.

Reply:
Perhaps by treating a zero duration as disabled. Zero is a common exception used for this and save a new parameter.